### PR TITLE
fix `Hammerhead is not initialized in the rewrited iframe with same-domain url in IE` (close #1325)

### LINKED ISSUE #1325: Hammerhead is not initialized in the rewrited iframe with same-domain url in IE
related with https://github.com/DevExpress/testcafe-hammerhead/pull/1312

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @miherlosev @AlexanderMoskovkin

### CODE REVIEW

File: test/client/fixtures/sandbox/iframe-test.js [hunk 1]
```diff
@@ -294,6 +294,19 @@ test("'body.appendChild' method works incorrectly in the particular case (GH-421
     checkIframes();
 });
 
+test('hammerhead must be initialized after the document with same-domain src is rewrited (GH-1325)', function () {
+    return createTestIframe({ src: getSameDomainPageUrl('../../data/iframe/simple-iframe.html') })
+        .then(function (iframe) {
+            var stringifiedNativeWriteFn = nativeMethods.documentWrite.toString();
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/iframe-test.js]: Let's check that hammerhead is properly initialized more strictly:
- set src attribute for link and check proxied url
- perform `document.write`
- again set src attribute for link and check proxied url

File: test/client/fixtures/sandbox/iframe-test.js [hunk 2]
```diff
@@ -294,6 +294,32 @@ test("'body.appendChild' method works incorrectly in the particular case (GH-421
     checkIframes();
 });
 
+test('hammerhead should be initialized after the document.write call in iframe with same-domain src (GH-1325)', function () {
+    return createTestIframe({ src: getSameDomainPageUrl('../../data/iframe/simple-iframe.html') })
+        .then(function (iframe) {
+            var stringifiedNativeWriteFn = nativeMethods.documentWrite.toString();
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/iframe-test.js]: Drop this line and appropriate assertions. You already check that iframe is properly initialized using link's src.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
https://github.com/DevExpress/testcafe-hammerhead/blob/c7cb85481c8a1a9461725cbf3629639d8ffd1b9d/test/client/fixtures/sandbox/node/document-write-test.js#L56-L60
https://github.com/DevExpress/testcafe-hammerhead/blob/c7cb85481c8a1a9461725cbf3629639d8ffd1b9d/test/client/fixtures/sandbox/node/document-write-test.js#L75-L81

**Comment by miherlosev** [comment]
Will be done in the context of an another task - https://github.com/DevExpress/testcafe-hammerhead/issues/1649

**Pull request closed without merging**